_data/hosting_providers.json

```
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "Snel.com",
+    "link": "https://snel.com/",
+    "category": "partial",
+    "tutorial": "",
```

For `partial`, we ask that you have a link to the documentation on how to enable SSL:

```suggestion
    "tutorial": "https://www.snel.com/support/set-up-ssl-for-my-website/",
```

---

I have made the changes accordingly. Once again thanks Alex!